LRAtsSDKSample/Internal/Extensions.swift

```diff
@@ -13,15 +13,24 @@ import LRAtsSDK
 extension LREnvelope {
     var stringRepresentation: String {
         var envelopeResultText: [String] = [String]()
-        if let rampID = envelope {
-            envelopeResultText.append("envelope:\n\(rampID)")
+        if let envelope {
+            envelopeResultText.append("envelope:\n\(envelope)")
         }
-        if let envelope24 = envelope24 {
+        if let envelope24 {
             envelopeResultText.append("envelope24:\n\(envelope24)")
         }
-        if let envelope25 = envelope25 {
+        if let envelope25 {
             envelopeResultText.append("envelope25:\n\(envelope25)")
         }
+        if let pairSegments {
+            envelopeResultText.append("pairSegments:\n\(pairSegments)")
```

Comment:
These are previous changes made in develop by levi9 team which were not merged to main